### DISCUSSION TITLE
Apple M1Max user: I think I’m going back to OBS27. Here’s why!

### DISCUSSION THREAD

**Discussion by nunowonder** [DistroAV/DistroAV#817] [General]
Só, I already wrote some solutions here on this project, on discussions, issues, etc, and I managed to have obs28 arm64 version, and ndi5 running on Apple Silicon (for M1 users). The usage of processor is great because finally OBS28 got it working on Apple Silicon, so the OBS can “read” this kind of processor (instead of emulating Intel).
What I noticed? (EDITED BECAUSE IT WAS ERROR FROM ME) I started having my connection/ transmission with problems, and I’m going back to obs27 just to check if the problem goes away. If it goes, I prefer to maintain old OBS even emulating my processing capabilities of my machine.
Do you guys know any problem about this or is it just me that realized that?

**Comment by Trouffman** [comment]
thanks for sharing, this type of issue is no longer an issue, plugin is working fine with apple silicon now
